### ISSUE TITLE
Add radio group/button example page

### ISSUE THREAD

**Issue by chrisglein** [microsoft/react-native-gallery#404]
Radio buttons are a commonly used control for desktop applications (in fact in the WinUI gallery it's categorized as "basic input"). There should be an example in Gallery of how to use them.
![image](https://github.com/microsoft/react-native-gallery/assets/26607885/cb965d9a-ece7-4cc3-8e10-9c24f89d9f61)

There are two main options that present themselves today:
- A) Use a JS-only control like https://github.com/thakurballary/react-native-radio-buttons-group
- B) Use react-native-xaml to access it directly

Would lean towards the former to start with since we already have a RNX page to cover the general concept, but it also wouldn't be bad to include within the page.

**Comment by Saadnajmi** [comment]
(drive by comment) If using a JS only control, the FURN Radiogroup might be an option

**Comment by chrisglein** [comment]
> (drive by comment) If using a JS only control, the FURN Radiogroup might be an option

I love drive by comments from Saad!
You're right. And it'd be more Fluent than the cross-platform module I used (see #411).  As is I provided 2 samples:
1. cross-platform module (not Fluent)
2. react-native-xaml Windows-specific (very Fluent)

FURN would solve both.

Buuuuuut.... to incorporate FURN into Gallery (Which I want to do! Will open an issue from this comment), I think we first need to address these:
- Required explicit importing of SVG (see https://github.com/microsoft/fluentui-react-native/issues/2613)
- Required switching to classic runtime (see https://github.com/microsoft/fluentui-react-native/issues/1891)
- Updates to the Windows visual theme (see [this override](https://github.com/chrisglein/artificial-chat/pull/116/files#diff-41ed9a259e808257d06e39c7a16b53685adf3e3fe5b73d59deb5ab59292f176fR109))
(I hit these when developing this: https://github.com/chrisglein/artificial-chat/pull/116)

Maybe we can partner up and fixing these and getting FURN in this Gallery app?
